entity-framework/core/modeling/owned-entities.md

```diff
@@ -11,9 +11,9 @@ EF Core allows you to model entity types that can only ever appear on navigation
 
 Owned entities are essentially a part of the owner and cannot exist without it, they are conceptually similar to [aggregates](https://martinfowler.com/bliki/DDD_Aggregate.html). This means that the owned entity is by definition on the dependent side of the relationship with the owner.
 
-## Explicit configuration
+## Configuring types as owned
 
-Owned entity types are never included by EF Core in the model by convention. You can use the `OwnsOne` method in `OnModelCreating` or annotate the type with `OwnedAttribute` to configure the type as an owned type.
+In most providers, entity types are never configured as owned by convention - you must explicitly use the `OwnsOne` method in `OnModelCreating` or annotate the type with `OwnedAttribute` to configure the type as owned. One exception to this is the Cosmos provider, which, as a document database, configures all related entity types as owned by default.
```

Comment:
```suggestion
In most providers, entity types are never configured as owned by convention - you must explicitly use the `OwnsOne` method in `OnModelCreating` or annotate the type with `OwnedAttribute` to configure the type as owned. One exception to this is the Cosmos provider, which, as a document database, configures all related entity types as owned by default.
```
```suggestion
In most providers, entity types are never configured as owned by convention - you must explicitly use the `OwnsOne` method in `OnModelCreating` or annotate the type with `OwnedAttribute` to configure the type as owned. The Azure Cosmos DB provider is an exception to this. Because Cosmos DB is a document database, the provider configures all related entity types as owned by default.
```